UPGRADE.md

```diff
@@ -3,6 +3,22 @@
 This document serves as a reference to upgrade your current BetterReflection installation if improvements, deprecations
 or backwards compatibility (BC) breakages occur.
 
+## 5.0.0
+
+### BC Breaks
+* Only PHP 8.0 is supported.
```

Comment:
Not a BC break 😁

Reply:
https://github.com/kukulich/BetterReflection/commit/cda8857c5767b260e61bef668eecc70a0e92d5af